modules/serving-transport-encryption-configuring-selfsigned-clusterissuer.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: PROCEDURE
+[id="serving-transport-encryption-configuring-selfsigned-clusterissuer_{context}"]
+= Configuring a SelfSigned ClusterIssuer
+
+[IMPORTANT]
+====
+For simplicity, this procedure uses a `SelfSigned` issuer as root certificate. However, be aware of the implications and limitations of this method documented in the "SelfSigned" section of the `cert-manager` documentation.
+
+If you run your company-specific Private Key Infrastructure (PKI), Red Hat recommends the CA issuer. Refer to the cert-manager documentation on CA issuers for more details. However, you can use any other issuer that allows signing of certificates as described in the "Choice of a certificate issuer" section.
+====
```

Comment:
I don't think that parts like this should be notes - it should be reformatted to be a proper abstract instead. Too many notes like this ends up being distracting and taking away from the distinction that having a note usually provides because they're overused.